### PR TITLE
June 2020 Update

### PR DESCRIPTION
**Pull request by givenscj**
June 2020 Update.

### CODE REVIEW

**Comment by shmiglan-arch** [comment]
14- Service Bus Queue(Path) not shown

**Comment by sha1painkiller** [comment]
It's targeting E2E IoT security solution. Is there a plan to include device-side security (for ex: ASC IoT Agent @IoT C SDK) for the workshop?

**Comment by shmiglan-arch** [comment]
1) Please replace IoT Provisioning Service with IoT Device Provisioning Service in the full document
2) Slide 14 as well as Hands-on MCW talks about routing data form IoT Hub to Service Bus, there is no service bus in the SA (Analytics).
3) Major - Though the Case Study talks about Security Center and Analytics use case, in implementation, what is shown is only the Security Center. The Analytics use case is not shown. SA (Analytics) is not demonstrated at all.
4) HOL step-by-step - Securing the IoT end-to-end.md- Task 3, Enable Azure Audit logging, One screenshot missing how to enable logs and link it to oilwells-logging-[YOUR INIT].

**Comment by DawnmarieDesJardins** [comment]
@givenscj 
Please see draft feedback on 7/8 from idbanham located in issue #10 
Thank you.

**Comment by cwtan7** [comment]
1) Azure Sphere has been mentioned and missing in the reference architecture.  Would be good to have the coverage of the design into the architectural recommendation.  This will definitely provide clear understanding / guidance on how to deploy sphere as a end-to-end security solution to strengthen the overall training centering "Security IoT E2E".

2) Would this training also consider Azure Sentinel since the material covers ASC to have a more comprehensive to better reflecting enterprise scenarios.

**Comment by givenscj** [comment]
@shmiglan-arch - I see where some disconnect is...a business preferred solution slide vs the security preferred solution slide. I will add a second slide that has security as the focus.

@sha1painkiller - you may not have run through the labs where agent/sdk is downloaded and installed?

@cwtan7 - I can add a 3rd slide with future state as Azure Sphere included.  It was request that be added a few iterations back but it wasn't quite fully baked so we didn't go into much detail.  We have another MCW that is focused on Azure Sentinel, not sure how much we'd want to hit on this here.

**Comment by givenscj** [comment]
I was able to go back in and review the comments in more depth and compare to content. It was not clear as to whether each of you were discussing the presentation aspect or the HOL aspect.  I added a solution slide for service bus to the presentation so that is represented now. 

Even though the labs start out with the solution diagrams, they simply try to focus on the security parts (Security the IoT end to end) even though there are multiple slices at the "solution" (such as business, security, analytics, processing).  I can certainly add in stream analytics and service bus to the labs, but I'm not sure that would be a valuable addition given the security focus of the MCW.  And if I did add, I would need to add something on the other side of each that would be security oriented (such as cog services anomaly detection or similar).  This would add considerable length to the labs and require some dev time.  I'm open to doing it, but wonder if the cost benefit is justifiable given the main focus of the MCW?

File: Hands-on lab/HOL step-by step - Securing the IoT end-to-end.md [hunk 1]
```diff
@@ -832,7 +952,7 @@ This exercise will evaluate the logs from when you enabled diagnostic logging on
 
 15. Select **OK**.
 
-16. On the **Configured actions** dialog, select **Done**.
+16. Select **OK**.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by step - Securing the IoT end-to-end.md:955]: Please check instructions & image.  Instructions 14, 15, and 16 are all Select OK. (3 OKs in a row?)
The image after instruction 14 doesn't have an OK in it....is this the right image?

**Comment by DawnmarieDesJardins** [comment]
@timahenning @givenscj 
Before the HOL has not been updated.  Date still reads March 2020 and it looks like Provisioning was taken out of the step-by-step but there is still mention of it in the Before the HOL.

**Comment by DawnmarieDesJardins** [comment]
@timahenning @givenscj 
Changes have been made to the trainer guide but no student guide updates (including date last reviewed...June 2020). Please update student guide.

**Comment by DawnmarieDesJardins** [comment]
@timahenning @givenscj 
PPT review - 
Slide 14 - Alt-text is "Envisioned Situation" need detailed alt-text please.
Slides 18 - 22 have no alt-text

**Comment by DawnmarieDesJardins** [comment]
Please review open issues #13 & #15  and let me know if they were addressed in this update and I can close them as complete when I merge this PR.

**Pull request merged**